### ISSUE TITLE
Cant go into any servers on Hexxit 2 anymore

### ISSUE THREAD

**Issue by Mrmikey2911** [xJon/Hexxit-II#225]
Im on java 8 and also on 64 bit.
I dont have any extra mods downloaded.

So im trying to get into Hexxit 2 , i can go into solo player , but not into multiplayer. I do have Java as in minecraft in general , i also have jave 8 and 64 bit when i run hexxit 2 , so does someone know how to fix this? i already retried deleting the launcher and downloading it again , but nothing seems to work. So can someone help me?

Kind Regards

Mrmikey2911

**Comment by xJon** [comment]
Have you migrated your Minecraft account into a Microsoft account?

**Comment by Eliseo3503** [comment]
this is also happening to me and i have migrated to a microsoft account and ive done everything and it isnt working is there a way to fix this

**Comment by xJon** [comment]
@Mrmikey2911 @Eliseo3503 I'd need more information from both of you to try to help - screenshots of the error or ideally client logs.

**Comment by Jedifakeboy** [comment]
![Screenshot (9)](https://github.com/xJon/Hexxit-II/assets/158706529/85ac0f99-66c1-4420-b4e5-8fe770645a2b)

**Comment by xJon** [comment]
@Jedifakeboy Do you have a legitimate Minecraft copy linked to a Microsoft account?

**Comment by Jedifakeboy** [comment]
> @Jedifakeboy Do you have a legitimate Minecraft copy linked to a Microsoft account?

yeah i do, my friend gifted it to me on this account a couple weeks back, i think it may have something to do with gifting because even the MC launcher doesn't say I have it. But I tried it on my friends account and I wasn't successful.

**Comment by xJon** [comment]
@Jedifakeboy Can you share your latest client log (during the issue occurrence) using https://mclo.gs?

**Comment by xJon** [comment]
Most likely an issue on Mojang's end: https://minecraftstatus.net/

**Comment by xJon** [comment]
Closing due to inactivity.
